bucket/pageedit.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.0.0",
+    "homepage": "https://github.com/Sigil-Ebook/PageEdit",
+    "description": "ePub XHTML Visual Editor.",
```

Comment:
```suggestion
    "description": "ePub XHTML Visual Editor",
```

Reply:
I copied the json from the sigil app. Should I fix that one as well?